locales/en/index.yml

```diff
@@ -11,7 +11,7 @@ global:
     reference: !include markdown_reference.md
     decodeError: Message can't be decoded
   app:
```

Comment:
the location of these two keys doesn't look good, they should both be under "profile" (the keys are organized per screen - the only exception is the keys for generic actions like ok/cancel/etc...)